### ISSUE TITLE
vmalert: add alert triggered timestamp to template variables.

### ISSUE THREAD

**Issue by laixintao** [VictoriaMetrics/VictoriaMetrics#2999]
**Is your feature request related to a problem? Please describe.**

I am trying to implement something like this: when an alert is fired, it not only just sends plain text messages, it also send a picture (chart image) describing what just happened.

**Describe the solution you'd like**

I can add something like this into alert's annotations:

```yaml
      - alert: Node_High_MemoryUsage
        expr: ...
        for: 1m
        labels:
          severity: page
        annotations:
          image:  ![](https://chart-service.vm.mydomain.com?renderExpr={{ $expr|quotesEscape|queryEscape }}&fireTimestamp={{ $fireTimestamp }}
```

This image will show serval images explains what is going on, for e.g. `cpu > 0.8 or memory > 0.6` , the image will show two charts, one for CPU compare to threshold 0.8 and the shows the memory chart compares 0.6. (spliting the alert rules expr into regular expression is based on your work of MetricsQL: https://github.com/laixintao/promqlpy )

So when the receiver opens this message(like email or slack), he/she will see the image rendered.

In order to render the chart with correct timerage, I need to know when is the alert triggered.

Also, it would be better if we can add `for` into template variables as well so I can render images time ranges for `$for * 60`, so that a larger `for` will get a wider chart.

**Describe alternatives you've considered**

I have tried to using vmalert's `-notifier.url` to trigger the image render, but it's hard to match the rendered image with the alerts in message.

**Additional context**

**Comment by laixintao** [comment]
@hagen1778 if it makes sense to you, I can give it a try, seems there is already `ActiveAt` variable there. WIP:https://github.com/VictoriaMetrics/VictoriaMetrics/pull/3000/files

**Comment by valyala** [comment]
FYI, the `$activeAt` template variable is available in `vmalert` starting from [v1.81.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases).
